youtrack-backup.pl

```diff
@@ -178,18 +171,32 @@ ()
 # https://www.backblaze.com/docs/cloud-storage-upload-files-with-the-cli
 # https://f000.backblazeb2.com/file/jsonwaterfalls/B2%20CLI%20Guide.pdf
 
-printf "%s: Uploading file to Backblaze B2 bucket $b2_bucket\n", tstamp() if (! $quiet);
+output_if_not_quiet("Uploading file to Backblaze B2 bucket $b2_bucket\n");
 my $tmpfile = File::Temp->new( TEMPLATE => '/tmp/XXXXXXXXXX' );
 my $cmd     = "b2 upload-file --noProgress $b2_bucket $download_path $file_name > $tmpfile";
-$result     = system($cmd);
-if ($result != 0) {
-	`cat $tmpfile`;
-	unlink($download_path);
-	die "Upload failed: $result\n";
+$result     = system $cmd;
+if ( $result != 0 ) {
+    `cat $tmpfile`;
+    unlink $download_path;
+    croak "Upload failed: $result\n";
 }
 
-printf "%s: Upload to Backblaze B2 bucket complete\n", tstamp() if (! $quiet);
+output_if_not_quiet("Upload to Backblaze B2 bucket complete\n");
 
-unlink($download_path);
+unlink $download_path;
 
-exit(0);
+exit 0;
+
+sub tstamp() {
+    return strftime '%H:%M:%S', gmtime;
+}
+
+sub output_if_not_quiet {
+    my ($output) = @_;
+
+    if ( !$quiet ) {
+        $time_stamped_output = sprintf "%s: %s", tstamp(), $output;
+        print $time_stamped_output;
```

Comment:
It seems odd to me to sprintf something to only print it the next line.

```suggestion
        printf "%s: %s", tstamp(), $output;
```